### PR TITLE
feat: update kafka topic provisioning config

### PR DESCRIPTION
**Pull request by kimxogus**
https://github.com/getsentry/snuba/commit/9790335825a087ecb6bba800ac323d3cc728cbbf
https://github.com/getsentry/snuba/commit/c1e30efa6ed0cd608ea14efa4270446b0c6b2b37

### CODE REVIEW

File: sentry/values.yaml [hunk 1]
```diff
@@ -1346,18 +1346,30 @@ kafka:
           "message.timestamp.type": LogAppendTime
       - name: event-replacements
       - name: snuba-commit-log
+        config:
+          "cleanup.policy": "compact.delete"
```
Comment (DanielMcAssey) [inline sentry/values.yaml]: Should be compact,delete (Note the comma instead of dot)

File: sentry/values.yaml [hunk 2]
```diff
@@ -1346,18 +1346,30 @@ kafka:
...
+          "min.compaction.lag.ms": "3600000"
       - name: cdc
       - name: transactions
         config:
           "message.timestamp.type": LogAppendTime
       - name: snuba-transactions-commit-log
+        config:
+          "cleanup.policy": "compact.delete"
```
Comment (DanielMcAssey) [inline sentry/values.yaml]: Should be compact,delete (Note the comma instead of dot)

File: sentry/values.yaml [hunk 3]
```diff
@@ -1346,18 +1346,30 @@ kafka:
...
+          "min.compaction.lag.ms": "3600000"
       - name: snuba-metrics
         config:
           "message.timestamp.type": LogAppendTime
       - name: outcomes
       - name: ingest-sessions
       - name: snuba-sessions-commit-log
+        config:
+          "cleanup.policy": "compact.delete"
```
Comment (DanielMcAssey) [inline sentry/values.yaml]: Should be compact,delete (Note the comma instead of dot)

File: sentry/values.yaml [hunk 4]
```diff
@@ -1346,18 +1346,30 @@ kafka:
...
+          "min.compaction.lag.ms": "3600000"
       - name: snuba-metrics-commit-log
+        config:
+          "cleanup.policy": "compact.delete"
```
Comment (DanielMcAssey) [inline sentry/values.yaml]: Should be compact,delete (Note the comma instead of dot)

File: sentry/values.yaml [hunk 5]
```diff
@@ -1385,12 +1397,24 @@ kafka:
         config:
           "message.timestamp.type": LogAppendTime
       - name: snuba-generic-metrics-sets-commit-log
+        config:
+          "cleanup.policy": "compact.delete"
```
Comment (DanielMcAssey) [inline sentry/values.yaml]: Should be compact,delete (Note the comma instead of dot)

File: sentry/values.yaml [hunk 6]
```diff
@@ -1385,12 +1397,24 @@ kafka:
...
+          "min.compaction.lag.ms": "3600000"
       - name: snuba-generic-metrics-distributions-commit-log
+        config:
+          "cleanup.policy": "compact.delete"
```
Comment (DanielMcAssey) [inline sentry/values.yaml]: Should be compact,delete (Note the comma instead of dot)

File: sentry/values.yaml [hunk 7]
```diff
@@ -1385,12 +1397,24 @@ kafka:
...
+          "min.compaction.lag.ms": "3600000"
       - name: snuba-generic-metrics-counters-commit-log
+        config:
+          "cleanup.policy": "compact.delete"
```
Comment (DanielMcAssey) [inline sentry/values.yaml]: Should be compact,delete (Note the comma instead of dot)

File: sentry/values.yaml [hunk 8]
```diff
@@ -1385,12 +1397,24 @@ kafka:
...
+          "min.compaction.lag.ms": "3600000"
       - name: generic-events
         config:
           "message.timestamp.type": LogAppendTime
       - name: snuba-generic-events-commit-log
+        config:
+          "cleanup.policy": "compact.delete"
```
Comment (DanielMcAssey) [inline sentry/values.yaml]: Should be compact,delete (Note the comma instead of dot)

**Comment by kimxogus** [comment]
@DanielMcAssey Thanks for the review, fixed all cleanup.policy configs

**Comment by kimxogus** [comment]
@DanielMcAssey Can you review this?

**Comment by DanielMcAssey** [comment]
Looks good! @Mokto could this be merged?

File: sentry/Chart.yaml [hunk 9]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: sentry
 description: A Helm chart for Kubernetes
 type: application
-version: 22.1.1
+version: 22.1.2
```
Comment (Mokto) [inline sentry/Chart.yaml]: Can you please remove that change?

Comment (kimxogus) [reply sentry/Chart.yaml]: @Mokto Removed chart version change

**Review by Mokto — APPROVED** [review]

**Pull request merged**